.cirrus.yml

```diff
@@ -0,0 +1,8 @@
+task:
```

Comment:
/cc @fkorotkov for a review of whether this is canonical usage of `.cirrus.yml`

Reply:
`matrix` looks invalid.

BTW name of this task will be `main`. If you want to change it you can either use `FOO_task` syntax or add `name: FOO` field to the `task`.

Reply:
Is there documentation on this? I found https://cirrus-ci.org/guide/writing-tasks/ but it's really vague and doesn't actually list all the possible directives and where they're allowed and what the syntax is and so on.

Reply:
There is `Task Naming` tip on this page. If your editor support JSON schemas from http://schemastore.org/json/ then you can get code assistance for `.cirrus.yml` files based on [`.cirrus.yml` schema](http://json.schemastore.org/cirrus).

Reply:
That's a start, but it seems incomplete. For example, the schema doesn't seem to say what fields are required (e.g. it doesn't seem to say that "image" is required for "container", though if you omit "image" from "container" the bot complains). It also doesn't answer more fundamental questions like "what does it mean to specify both 'container' and 'windows_container'" or "what images can I use for a 'container'". :-(

Reply:
Yeah, it's not possible to describe with the JSON schema standard. Create a feature request: https://github.com/cirruslabs/cirrus-ci-docs/issues/334